_data/versions.yml

```diff
@@ -1,3 +1,57 @@
+v2.3:
+- title: v2.3.0-rc1
+  note: |
+    12 June 2017
+
+    ### Changes / New Features
+
+    - Various improvements which reduce Felix CPU usage.
+    - Introduces the Calico fanout agent, Typha, for better performance when using the Kubernetes datastore driver.
+    - Calico now implements the networking.k8s.io/NetworkPolicy API semantics when using the Kubernetes datastore driver.
+      - **Note**: Calico with etcd continutes to implement the extensions/v1beta1 NetworkPolicy API semantics - this change has no effect on etcd based Calico deployments. 
+      - **Note**: This represents a change in how existing Kubenretes NetworkPolicies are enforced by Calico.  To maintain existing behavior when upgrading, follow these steps:
+              - In Namespaces that previously did not have the "DefaultDeny" annotation, you should delete any existing NetworkPolicy objects.
+              - In Namespaces that previously did have the "DefaultDeny" annotation, you can create the equivalent semantics by creating a NetworkPolicy that selects all pods 
```

Comment:
also can we point to k8s release notes for this so people don't get confused thinking this is with calico policies? because this is kind of a big change